canonical/maas-ui
# Side identity - MAAS theme live preview

## Done

- Added a context for tracking the current MAAS theme
- Allowed users to preview a theme by selecting the radio button
- Added a "Cancel" button should a user change their mind

## QA
- Resets the MAAS theme to their existing choice if they refresh the page or navigate elsewhere

### QA steps

- Log in on the PR preview
- Preview a theme by clicking its radio button
- Save the theme and ensure it sticks if you refresh/navigate
- Preview a different theme
- Ensure it reverts back if you:
  - Hit "Cancel"
  - Navigate to another page
  - Refresh the page 

## Fixes

Fixes: https://github.com/canonical/app-tribe/issues/1133 .
